### DISCUSSION TITLE
How can I remove "Packages" and "Releases" from my project's front page?

### DISCUSSION THREAD

**Discussion by tripleee** [community/community#21653] [New to GitHub]
<p>On my project’s page on GitHub, on the right-hand side, just below the license information, I see “<strong>Releases</strong> No releases published - Create a new release; <strong>Packages</strong> No packages published - Publish your first package”.  I currently have no use for these features, and would like to turn them off just to reduce noise.</p>
<p>I am pretty sure I was asked about this recently, but at the time I accepted, and now I can find no place where I can disable these.</p>
<p>I <em>think</em> I received the question when I set up GitHub Actions for my project, but I also see these headlines on the pages of projects which don’t have any GitHub Actions.</p>
<p>Is there a way to turn these off? Where should I look?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p><img src="https://user-images.githubusercontent.com/47092464/181066024-ba5b990f-f855-4a08-ad47-0c4cead9943e.png" alt="image" data-base62-sha1="6zXCE4BWudxcJmEut18M8YUBSKQ" width="443" height="315"></p>
<p>click the gear button</p>
<p>then you’ll see this, uncheck them</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/f/1f290552ef638ca47cedb6877bce77a795c232c8.png" data-download-href="/uploads/short-url/4rEGFzOyIfjdh4P8UFBno9385uw.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181066025-e2942572-7546-475d-8631-a330f461efdb.png" alt="image" data-base62-sha1="4rEGFzOyIfjdh4P8UFBno9385uw" width="400" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/f/1f290552ef638ca47cedb6877bce77a795c232c8_2_400x500.png, https://aws1.discourse-cdn.com/github/original/3X/1/f/1f290552ef638ca47cedb6877bce77a795c232c8.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/f/1f290552ef638ca47cedb6877bce77a795c232c8.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/f/1f290552ef638ca47cedb6877bce77a795c232c8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">562×702 34.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Reply by aflacake** [reply]
This menu is not found on mobile. Can find with a computer/laptop

**Comment by ghost** [comment]
Even when I disable them, they still appear. @jdevstatic

**Reply by ghost** [reply]
I can report the same issue on my side. I haven't been able to disable releases, packages or environments on new repositories in a while. I can't pinpoint exactly when this started, but it doesn't seem to be caused by any of the feature previews, as the behavior is the same with or without them.

**Reply by ghost** [reply]
So? Is there a fix in the works? It's quite peculiar.

**Reply by ghost** [reply]
I haven't got a clue, I looked this up today and this was the only thread I could find on the issue after a short Google search. I dove deeper and it seems that the issue has been reported [several times](https://github.com/orgs/community/discussions?discussions_q=packages+releases+environments) already, so the folks at GitHub must already be aware of it. #62025 is the most active thread on this subject, in case you wish to subscribe to it.

**Reply by ghost** [reply]
@smilorent Thank you, I _did_ subscribe. Also, thanks for linking the thread. 🙈

**Comment by camlafit** [comment]
Hello

Is it possible to disable it for an organization ? 
New repository creation display "package" block. As package provide only a small list in some case this feature is not adapted and will be always empty.

Thanks a lot

**Reply by AuthFailed** [reply]
Hi! Did you find solution to this?

**Reply by camlafit** [reply]
I've only found a workaround, I host my own "à la" github forge " with my own gitea/forgejo instance.

**Comment by dariocurr** [comment]
Is there a way to achieve that using Github API?
